test/code/plugins/oms_common_test.rb

```diff
@@ -94,6 +100,256 @@ def teardown
       "OSAlias=UniversalR\n" \
       "OSManufacturer=Central Logistics GmbH\n"
 
+    # Begin:  Unit Tests for some simple methods.
+    # May best be put in another file later.
+
+    def test_clean_hostname_string
+      hn = ' www.five.com:333     '
+      b = Common.clean_hostname_string(hn)
+      expected = 'www.five.com'
+      assert_equal(b,expected,"method clean_hostname_string should return '#{expected}'.")
+      hn = '        microsoft     '
+      b = Common.clean_hostname_string(hn)
+      expected = 'microsoft'
+      assert_equal(b,expected,"method clean_hostname_string should return '#{expected}'.")
+      hn = expected
+      b = Common.clean_hostname_string(hn)
+      assert_equal(b,expected,"method clean_hostname_string should return '#{expected}'.")
```

Comment:
style nit: add spaces to all of your parameter lists

Reply:
Sure.